blockstore/src/main/java/brooklyn/location/blockstore/BlockDeviceOptions.java

```diff
@@ -12,7 +12,7 @@
     private String name;
     private String zone;
     private Map<String, String> tags = Maps.newHashMap();
-    private int sizeInGb;
+    private Number sizeInGb;
```

Comment:
Agree, but you can convert it to an `int` on line [34](https://github.com/cloudsoft/brooklyn-blockstore/pull/3/files#diff-0fd1dffb823987019aee58245a99910cR34) (by casting).